packages/core/src/lib/session.ts

```diff
@@ -0,0 +1,18 @@
+import { Context } from 'koa';
+import { InteractionResults, Provider } from 'oidc-provider';
+
+// TODO: change this after frontend is ready.
+// Should combine baseUrl(domain) from database with a 'callback' endpoint.
+export const connectorRedirectUrl = 'https://logto.dev/callback';
```

Comment:
looks like this should be a fixed route based on current Logto domain which has nothing related to frontend?

Reply:
removed improper comments.

Reply:
This is a redirect URI after a social connector authorization process.
It's a front-end path that has not been determined, needs to be changed after the front-end implementation.